tests/unit/test_classifier.py

```
@@ -226,3 +226,49 @@ def test_classification_result_non_fallback_normal_score() -> None:
+def test_classifier_explicit_procedural_tag() -> None:
+    """[🕒 Procedural] タグによる分類。"""
+    raw = _make_raw("[🕒 Procedural]\nセットアップ方法を説明します。")
+    classifier = Classifier()
+    result = classifier.classify(raw)
+
+    assert result.memory_type == MemoryType.PROCEDURAL
+    assert result.confidence == pytest.approx(1.0)
+
+
+def test_classifier_explicit_tag_overrides_other_patterns() -> None:
+    """明示的なタグは、中身の内容（例：過去形）よりも優先される。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)